src/GafferBindings/CompoundNumericPlugBinding.cpp

```diff
@@ -54,6 +56,23 @@ namespace
 {
 
 template<typename T>
+std::string maskedCompoundNumericPlugRepr( const T *plug, unsigned flagsMask )
+{
+	std::string extraArgs = "";
+
+	const IECore::GeometricData::Interpretation interpretation = plug->interpretation();
+	if( interpretation != IECore::GeometricData::None && interpretation != IECore::GeometricData::Color )
```

Comment:
Looks like you got the same suggestion in before I finished my reply :). If you wanted, we could make the default be GeometricData::Color for Color3f plug by adding a type traits class of some sort. Or we could argue that Color3fPlug shouldn't have an interpretation anyway, and we were wrong to make all CompoundNumericPlugs have interpretations, and instead introduce a separate template class to use for V3fPlug etc. I'm more inclined to go with Matti's proposed solution though - this feature was always going to be a bit esoteric, so I'd prefer it to have as little impact on the API as possible.